src/renderer/prefComponents/general/index.vue

```diff
@@ -2,87 +2,119 @@
   <div class="pref-general">
     <h4>General</h4>
     <compound>
-    <template #head>
-    <bool
-      description="Automatically save document changes"
-      :bool="autoSave"
-      :onChange="value => onSelectChange('autoSave', value)"
-    ></bool>
-    </template>
-    <template #children>
-    <range
-      description="Delay following document edit before automatically saving"
-      :value="autoSaveDelay"
-      :min="1000"
-      :max="10000"
-      unit="ms"
-      :step="100"
-      :onChange="value => onSelectChange('autoSaveDelay', value)"
-    ></range>
-    </template>
+      <template #head>
+        <h6 class="title">Auto Save:</h6>
+      </template>
+      <template #children>
+        <bool
+          description="Automatically save document changes"
+          :bool="autoSave"
+          :onChange="value => onSelectChange('autoSave', value)"
+        ></bool>
+        <range
+          description="Delay following document edit before automatically saving"
+          :value="autoSaveDelay"
+          :min="1000"
+          :max="10000"
+          unit="ms"
+          :step="100"
+          :onChange="value => onSelectChange('autoSaveDelay', value)"
+        ></range>
+      </template>
+    </compound>
+
+    <compound>
+      <template #head>
+        <h6 class="title">Window:</h6>
+      </template>
+      <template #children>
+        <cur-select
+          v-if="!isOsx"
+          description="Title bar style"
+          notes="Requires restart."
+          :value="titleBarStyle"
+          :options="titleBarStyleOptions"
+          :onChange="value => onSelectChange('titleBarStyle', value)"
+        ></cur-select>
+        <bool
+          description="Hide scrollbars"
+          :bool="hideScrollbar"
+          :onChange="value => onSelectChange('hideScrollbar', value)"
+        ></bool>
+        <bool
+          description="Open files in new window"
+          :bool="openFilesInNewWindow"
+          :onChange="value => onSelectChange('openFilesInNewWindow', value)"
+        ></bool>
+        <bool
+          description="Open folders in new window"
+          :bool="openFolderInNewWindow"
+          :onChange="value => onSelectChange('openFolderInNewWindow', value)"
+        ></bool>
+      </template>
+    </compound>
+
+    <compound>
+      <template #head>
+        <h6 class="title">Sidebar:</h6>
+      </template>
+      <template #children>
+        <bool
+          description="Wrap text in table of contents"
+          :bool="wordWrapInToc"
+          :onChange="value => onSelectChange('wordWrapInToc', value)"
+        ></bool>
+
+        <!-- TODO: The description is very bad and the entry isn't used by the editor. -->
+        <cur-select
```

Comment:
@Jocs Whats the meaning of this entry? It's never used anywhere. Sorting the sidebar by the given options?

Reply:
It selects whether to hide the native window decorations and replace them with items rendered internally by the application.

Reply:
If you want to change it, perhaps you would consider a boolean entry called "Use native window decorations".

Reply:
(As a separate matter, it seems there is no prompt to restart the application after the setting changes. Such a feature would be convenient, but not necessarily  so desirable without session management.)

Reply:
I have an idea of moving the items  from the *General* section into other sections. This option and some others might be moved into the same section as theme selection, which might be renamed *Appearance*.  Other items might be moved to *Editor*.

Reply:
The settings entry is about `fileSortBy` and not window decorations. Thats why I asks because the entry is never used by us and was disabled before.

Reply:
Yes, sorry, my mistake.

Reply:
>Sorting the sidebar by the given options?

Yes, and not implemented?

Reply:
It was disabled in preferences and nowhere used if you search the property.